jsonrpc/backend/django.py

```diff
@@ -1,7 +1,12 @@
 from __future__ import absolute_import
 
 from django.views.decorators.csrf import csrf_exempt
-from django.conf.urls import url
+
+try:
+    from django.conf.urls import url  # Django <4.0
+except ImportError:
+    from from django.urls import re_path as url  # Django >=4.0
```

Comment:
Doubled `from` 🤦🏻

Reply:
Thank you for flagging. Fixed and released `1.15.0`.